content/en/docs/instrumentation/java/automatic/annotations.md

```diff
@@ -3,6 +3,7 @@ title: Annotations
 description: Using instrumentation annotations with a Java agent.
 aliases: [/docs/instrumentation/java/annotations]
 weight: 4
+javaAnnotationsVersion: 1.17.0-alpha
```

Comment:
We might as well bump it right now; the new instrumentation version was released yesterday 😄 
```suggestion
javaAnnotationsVersion: 1.18.0-alpha
```